tests/lifecycle/lifehelper/lifehelper.go

```diff
@@ -207,6 +209,37 @@ func EnableMasterScheduling(scheduleable bool) error {
 	return err
 }
 
+func DefineDaemonSet(namespace string, image string, label map[string]string, name string) *v1.DaemonSet {
+	return &v1.DaemonSet{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      name,
+			Namespace: namespace},
+		Spec: v1.DaemonSetSpec{
+			Selector: &metav1.LabelSelector{
+				MatchLabels: label,
+			},
+			Template: corev1.PodTemplateSpec{
+				ObjectMeta: metav1.ObjectMeta{
+					Name:   "testpod-",
+					Labels: label,
+				},
+				Spec: corev1.PodSpec{
+					TerminationGracePeriodSeconds: pointer.Int64Ptr(0),
+					Containers: []corev1.Container{
+						{
+							Name:    "test",
+							Image:   image,
+							Command: []string{"/bin/bash", "-c", "sleep INF"}}}}}}}
+}
+
+func DefineDaemonSetWithImagePullPolicy(name string, image string, pullPolicy corev1.PullPolicy) *v1.DaemonSet {
+	daemonSet := daemonset.RedefineWithImagePullPolicy(
```

Comment:
You can simplify using return on this line

Reply:
Fixed, thanks.